nemo_skills/code_execution/utils.py

```diff
@@ -16,16 +16,27 @@
 from typing import Dict, Tuple
 
 
-def format_code_output(execution_dict: Dict[str, str], code_output_begin: str, code_output_end: str):
+def format_code_output(
+    execution_dict: Dict[str, str], code_output_begin: str, code_output_end: str, code_output_format: str = 'llama'
+):
     """Formatting code output to be displayed as an llm expects it."""
-    output = execution_dict["process_status"]
-    if execution_dict['stdout']:
-        output += f"\n[stdout]\n{execution_dict['stdout']}\n[/stdout]"
-    if execution_dict['stderr']:
-        output += f"\n[stderr]\n{execution_dict['stderr']}\n[/stderr]"
+    if code_output_format == 'llama':
+        output = execution_dict["process_status"]
+        if execution_dict['stdout']:
+            output += f"\n[stdout]\n{execution_dict['stdout']}\n[/stdout]"
+        if execution_dict['stderr']:
+            output += f"\n[stderr]\n{execution_dict['stderr']}\n[/stderr]"
+        output = f"{code_output_begin}\n\n{output}{code_output_end}\n\n"
+    elif code_output_format == 'openmathinstruct':
+        output = f"\n{execution_dict['stdout']}\n" if execution_dict['stdout'] else f"\n{execution_dict['stderr']}\n"
```

Comment:
We should do concatenation of stdout and stderr here and below for qwen

Reply:
Actually let's remove openmathinstruct format, it's almost the same as qwen anyway, right?